sys/net/gnrc/routing/rpl/gnrc_rpl.c

```diff
@@ -149,17 +157,17 @@ static void _receive(gnrc_pktsnip_t *icmpv6)
         case GNRC_RPL_ICMPV6_CODE_DIO:
             DEBUG("RPL: DIO received\n");
             gnrc_rpl_recv_DIO((gnrc_rpl_dio_t *)(icmpv6_hdr + 1), iface, &ipv6_hdr->src,
```

Comment:
I wonder if we could just pass the `&ipv6_hdr` to the functions and access `->src`, `->dst` and `->len` inside the function.